### PR TITLE
Galaxy Lawyer Suit

### PR DESCRIPTION
**Pull request by bobbah**
## About The Pull Request

Ports the [corporate liaison galaxy suit from TGMC](https://github.com/tgstation/TerraGov-Marine-Corps/pull/668). Requested by Grantteen. Object sprites by @Twaticus.

Washing a lawyer suit with a bluespace crystal or a telecrystal will dye the suit into an exciting galaxy suit variant. This also adds the DYE_SYNDICATE dye key to telecrystals, could be used for future PRs.

image

image

image

image

## Why It's Good For The Game

Adds variety to the horrors of washing machine code.

## Changelog
:cl: bobbahbrown
add: Washing a lawyer's jumpsuit with a bluespace crystal or telecrystal will dye the suit into an exciting galactic suit!
/:cl:

### CODE REVIEW

**Comment by Mey-Ha-Zah** [comment]
Dat commit message

**Comment by Tharcoonvagh** [comment]
im not a wizard hagrid you fat oaf I AM NOT A FUCKING WIZARD!

**Comment by Fhaxaris** [comment]
now all that is left for us to do is make it work like the armor that teleports you before you get hit.

**Comment by fludd12** [comment]
🤔 did you manage to get that commit text by translating the Entire First Chapter of HP into spanish and then back to english

**Comment by Shapsy** [comment]
Should make a brain with the same cosmic space effect and put it in the arcade machines so assistants can brag about their galaxy brains

**Comment by Shadowflame909** [comment]
@Fhaxaris 

Galaxy armor should have punches phase through you

File: code/modules/clothing/under/jobs/civilian/civilian.dm [hunk 1]
```diff
@@ -208,3 +214,17 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
+
+/obj/item/clothing/under/rank/civilian/lawyer/galaxy
```
Comment (SpaceManiac) [inline code/modules/clothing/under/jobs/civilian/civilian.dm]: I would prefer if we avoided "base" types that aren't actually useful - rather one of blue or red is the base outright

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/jobs/civilian/civilian.dm (hunk 1 above)
Comment (bobbahbrown) [reply code/modules/clothing/under/jobs/civilian/civilian.dm]: fixed

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**